libraries/linters/usfm_linter.py

```diff
@@ -82,9 +82,9 @@ def parse_usfm_text(self, sub_path, file_name, book_text, book_full_name, book_c
                 self.found_books.append(book_code)
 
             if len(errors):
-                prefix = "File '{0}' book code '{1}': ".format(sub_path, book_code)
                 for error in errors:
-                    self.log.warning(prefix + error)
+                    parts = error.split( ":" )
+                    self.log.warning( "{0} {1} - {2}".format(book_code, " ".join( parts[1:] ), parts[0] ))
```

Comment:
should have a sanity check here in case there is no ':' in the error message.

Reply:
Also, my IDE linter always complains about these spaces after the parentheses. Is it possible to remove the spaces? I know, a bit picky, but really prefer the code to be the same as above and below.